Quantum-Control-Applications/Superconducting/Single Fixed Transmon/Use Case 3 - DRAG calibration/README.md

```diff
@@ -0,0 +1,116 @@
+discrimination.
+
+### DRAG coefficient calibration with `x180` `-x180` pulses
+
+In this step we generate a 2D plot where the x-axis is `number of (x180)(-x180)` pulses
+applied, and the y-axis is the `DRAG coefficient`. By applying a train of pulses, the
+sequence accumulates more and more population in the second excited state with each
+extra pair of `x180` `-x180`, thus we gain sensitivity to pinpoint the DRAG coefficient.
+The pattern generated in the image below resembles a `Chevron-Ramsey`
+experiment. We can identify that the center line of the `Chevron-Ramsey` pattern is away from the zero.
+The figure below the 2D plot shows line cuts at pulse trains with `1`, `6`, and `11` iterations. The minima of these parabolas narrows as the number of iterations in the train of pulses
+increases. We fit quadratic functions to each of these experimental lines and find that for `iterations = 11`,
+the minimum is at `0.081 +/- 0.008`. Note that as the minima gets narrower we need to increase
+the granularity of the data points to accurately capture the experimental behavior. The program corresponding to this sequence can be found
+in `Google_DRAG.py`. The pulse train used for this experiment is a 2D extension of the one listed in Ref. [1].
+
+![Google_DRAg](Google_DRAG.png)
+![Google_linecuts](Google_DRAG_linecuts.png)
+
+### DRAG coefficient calibration with `x180y90` and `y180x90` pulses
+
+The figure below shows two lines that intersect at the value of the calibrated `DRAG coefficient`.
+Unlike the previous method, these sequences are made of `x180y90` and `y180x90` pulses. The resulting
+population is plotted against the DRAG coefficient. By fitting the lines and computing the x-value
+at which the lines intersect, from the fitted formulas, we can precisely derive the `DRAG coefficient` [2]. The derived
+DRAG value is `0.117 +/- 0.003`.
+
+It is relevant to highlight that the two methods we have just listed to find the `DRAG coefficient` give out
+different results and that they do not overlap within their respective uncertainties. For the rest 
+of the use case we arbitrarily choose `0.117 +/- 0.003` for the `DRAG coefficient`.
+
+
+![Reeds_drag](Reeds_DRAG.png)
+
+### AC Stark-shift correction
+
+Leakage out of computational space detunes the ground-to-excited state transition 
+during the duration of pulses [1]. With the following protocol it is possible to calibrate the 
+amount of detuning to compensate for the AC stark shift inflicted by the leaked population. Having fixed
+the DRAG coefficient we now sweep over the detuning applied to the pulse envelope; note that
+during this experiment we keep the `IF` frequency fixed as derived from a Ramsey experiment. The 2D
+plot below displays a `Chevron-Ramsey` pattern, centered at negative values away from `0 Hz` detuning. In the
+figure below the 2D plot, we did a vertical linecut at the train pulse with `20 iterations`
```

Comment:
Since you need to open a new QM for each detuning maybe it is to save some time and just run the experiment once with the desired detuning, is it correct?